public/OutlawDerby2024.md

```diff
@@ -0,0 +1,65 @@
+![outlawlogo]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby lOGO.png")
+
+
+# 2024 PACK 306 OUTLAW DERBY
+
+
+
+* Are you a former scout that loved participating in the Pinewood Derby?
+* Did you help your child build their Pinewood Derby car but you wish you could raced your own car?
+* Are you a racing fan that just wants to compete and win a prize?
+* Are you a local business owner that is looking for a fun different way to promote your business?
+
+
+
+Then the Outlaw Derby is your chance to build your own car and race. The best part is there are very little rules. There are two ways to enter, as a business sponsor or as an individual.
+
+
+
+---
+
+
+
+* What: "No" rules pinewood derby open to scouts, adults, members of the community, and local businesses
+* When: Saturday May 11th 1-4p
+* Where: CPC Pavilion, 1400 Frederick Road, Catonsville
+* Why: Raise funds for the pack, support local scouting, and most importantly to share the fun with our community!
+* More Info: cub306outlawderby@gmail.com
+
+## Local Business Entry (Sponsor a Scout): $50
+
+
+
+* A scout will build & race your car.
+* Your business name announced every time your car races.
+* Your car and business name featured on Pack Facebook page.
+* Announced in Pack-wide emails.
+
+
+
+## Individual Entry: $20
+
+
+
+* 1 PWD car kit
+* Registration
+
+
+
+### [Click here to register](https://cub306.square.site/outlaw-derby-2024)
+![outlaw2024qr]("C:\Users\brian\Documents\GitHub\cub306.org\public\images\outlaw derby qr code.png")
+
+
+
+## Car Creation Guidelines
+**For safety purposes and to protect the race track from damage, cars:**
+
+1) May not interfere with any other cars in the race;
```

Comment:
does ) work for markdown? I guess not. if you switch to '.' then you will get the list styles.

Reply:
i just copied  and pasted it.